src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -50,21 +50,92 @@ uct_cuda_ipc_mem_add_reg(void *addr, uct_cuda_ipc_memh_t *memh,
 {
     uct_cuda_ipc_lkey_t *key;
     ucs_status_t status;
+#if HAVE_CUDA_FABRIC
+    CUmemAllocationHandleType fabric_type = CU_MEM_HANDLE_TYPE_FABRIC;
```

Comment:
initialized vars should be first. maybe can avoid this init?

Reply:
Ok will directly use the enum and remove the variable altogether.